.github/workflows/publish_images.yaml

```diff
@@ -3,35 +3,36 @@ name: Build and publish OCI Images
 on:
   push:
     branches:
-      - master
+      - latest
   schedule:
     # * is a special character in YAML so you have to quote this string
     - cron: '0 1 * * *'
   workflow_dispatch:
     inputs:
       pulpcore:
-        description: 'Pulpcore Y-release Version (e.g. 3.7)'
+        description: 'Release branch (e.g. 3.7)'
         required: true
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.pulpcore }}
```

Comment:
I'm hoping that if this is not set, it'll default to the default branch (latest).

Reply:
this is the only thing I'm really unsure of, but let's test it"